src/secrets.js

```diff
@@ -180,4 +200,36 @@ function exportSecretToOutput(variableName, secretValue, exportSecretsToOutputs,
     }
 }
 
+function exportJsonFields(parsedJson, secretName, prefix, exportSecretsToOutputs, exportSecretsToEnvironment) {
+    if (prefix === undefined) {
```

Comment:
What about other values like `null`? do we need to handle them?

Reply:
Fixed